torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -132,10 +132,43 @@ FunctionParameter::FunctionParameter(const std::string& fmt, bool keyword_only)
   }
 }
 
-bool FunctionParameter::check(PyObject* obj) {
+static auto get_tensor_torch_function() -> PyObject*
```

Comment:
we lost the comment stating the precondition for the function. The assert is too indirect, since `PyObject_GetAttrString` explicitly is defined to return null on failure

Reply:
It turns out that this function is no longer used by anything so I'm just going to remove it.